vmdb/spec/models/host_spec.rb

```diff
@@ -433,6 +433,32 @@ def should_only_scan_in_its_zone(zone)
     end
   end
 
+  context "get_ports" do
+    before(:each) do
+      @zone1 = FactoryGirl.create(:small_environment)
```

Comment:
Do we need a whole small environment to test this?  Those are pretty big and it just looks like you need a Host.